pkg/pb/datarepair.proto

```diff
@@ -10,4 +10,14 @@ package repair;
 message InjuredSegment {
     string path = 1;
     repeated int32 lost_pieces = 2;
+    repeated int32 healthy_pieces = 3;
 }
+
+//IdentifyRequest is a request message for the identifyInjuredSegments private method
+message IdentifyRequest {
+    string prefix = 1;
+    string first = 2;
+    bool recurse = 3;
+    bool reverse = 4;
```

Comment:
Are all these really required for `Identify`?

Reply:
not necessarily, i wasnt sure what we would need eventually.